configs/baremetal.h

```diff
@@ -45,7 +45,7 @@
 #define MBEDTLS_PK_C
 #define MBEDTLS_PK_PARSE_C
 
-#define MBEDTLS_ENTROPY_MAX_SOURCES 1
+#define MBEDTLS_ENTROPY_MAX_SOURCES 3
```

Comment:
This should be kept same and the overriding value of 3 should be changed to `baremetal_test.h` that is the file used for tests and it overrides the usage of `MBEDTLS_TIMING_C` and `MBEDTLS_NO_PLATFORM_ENTROPY` from `baremetal.h`

Reply:
nice, I did not know that. Will fix.